build.sbt

```diff
@@ -263,10 +278,13 @@ lazy val examples = project
       "de.heikoseeberger"            %% "akka-http-circe"               % "1.32.0",
       "com.softwaremill.sttp.client" %% "async-http-client-backend-zio" % sttpVersion,
       "com.softwaremill.sttp.tapir"  %% "tapir-json-circe"              % tapirVersion,
-      "io.circe"                     %% "circe-generic"                 % circeVersion
+      "io.circe"                     %% "circe-generic"                 % circeVersion,
+      "com.typesafe.play"            %% "play-akka-http-server"         % playVersion,
+      "com.typesafe.akka"            %% "akka-actor-typed"              % akkaVersion,
```

Comment:
I had to add both `akka-actor-typed` and `akka-slf4j` dependencies to avoid the error below:
`
java.lang.IllegalStateException: You are using version 2.6.5 of Akka, but it appears you (perhaps indirectly) also depend on older versions of related artifacts. You can solve this by adding an explicit dependency on version 2.6.5 of the [akka-actor-typed, akka-slf4j] artifacts to your project. See also: https://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#mixed-versioning-is-not-allowed
`

Reply:
It's caused by `akka-http-play-json` that uses an older version of Akka but I wonder why we need that one, see my other comment.

Reply:
I guess those 2 aren't needed anymore?